docs/internals/howto-release-django.txt

```diff
@@ -159,16 +165,18 @@ OK, this is the fun part, where we actually push out a release!
    checkout security/1.5.x; git rebase stable/1.5.x``) and then switch back and
    do the merge. Make sure the commit message for each security fix explains
    that the commit is a security fix and that an announcement will follow
-   (`example security commit`__)
+   (`example security commit`__).
 
    __ https://github.com/django/django/commit/3ef4bbf495cc6c061789132e3d50a8231a89406b
 
-#. Update version numbers for the release. This has to happen in three
-   places: ``django/__init__.py``, ``docs/conf.py``, and ``setup.py``.
+#. Update the version number in ``django/__init__.py`` for the release.
    Please see `notes on setting the VERSION tuple`_ below for details
-   on ``VERSION``. Here's `an example commit updating version numbers`__
+   on ``VERSION``.
 
-   __ https://github.com/django/django/commit/18d920ea4839fb54f9d2a5dcb555b6a5666ee469
+   In 1.4, the version number in ``docs/conf.py`` and ``setup.py`` should also
+   be updated. Here's `an example commit updating version numbers`__ for that.
+
+   __ https://github.com/django/django/commit/592187e11b934f83153133cd5b3a246a881359e7
 
 #. For a major version release, remove the ``UNDER DEVELOPMENT`` header at the
    top of the release notes and add the release date on the next line. For a
```

Comment:
I also think we should move up the release notes changes so that it's step two, as it involves backward/forward porting. After that, the rest of the steps are done on only one branch.